README.md

```diff
@@ -75,7 +75,111 @@ More examples, as well as an explanation of input/output are available in [offic
 
 ### Using command line
 
-TODO: add this section.
+#### Install `extra-model`
+
+First, install `extra-model` via pip:
+
+```bash
+pip install extra-model
+```
+
+#### Downloading Embeddings
+
+Next, run the following to download and set up the required embeddings (we use [Glove](https://nlp.stanford.edu/projects/glove/) from Stanford in this project):
+
+```bash
+extra-model-setup
+```
+
+The embeddings will be downloaded, unzipped and formatted into a space-efficient format. By default, files will be saved in `/embeddings`. You can set another directory by providing it as an argument when running `extra-model-setup` like so:
+
+```bash
+extra-model-setup /path/to/store/embeddings
```

Comment:
I wonder if that works, actually. I'm fairly certain that embeddings can only be in `./embeddings` since we don't expose a way to change location with `run()`.

Reply:
Indeed. So this _is_ exposed in `_cli.py` [here](https://github.com/wayfair-incubator/extra-model/blob/main/extra_model/_cli.py#L54) but not in `_run.py`. I'll remove that arg for now.